### PR TITLE
Fix slow TestUsedClusterCancel test

### PR DESCRIPTION
**Pull request by alex-yust**
## Motivation and Context

#1691 

## How Has This Been Tested?

- [x] Covered by existing integration testing
- [ ] Added integration testing to cover
- [ ] Tested locally
- [ ] Have not tested

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

**Comment by denis-tingajkin** [comment]
`TestUsedClusterCancel` still most slow test (took 17.88 seconds). Can we speed up it?

File: test/cloudtest/pkg/tests/shell_provider_test.go [hunk 1]
```diff
@@ -291,7 +292,7 @@ func TestUsedClusterCancel(t *testing.T) {
 	testConfig.ConfigRoot = tmpDir
 	createProvider(testConfig, "a_provider")
 	p2 := createProvider(testConfig, "b_provider")
-	p2.TestDelay = 15
+	p2.TestDelay = 5
```
Comment (denis-tingajkin) [inline test/cloudtest/pkg/tests/shell_provider_test.go]: why it is 5? Can we use some lower value?

**Comment by alex-yust** [comment]
> `TestUsedClusterCancel` still most slow test (took 17.88 seconds). Can we speed up it?

Ok.. it is now 10-12s
https://109811-127937836-gh.circle-artifacts.com/0/home/circleci/junit/unit-tests.xml

File: test/cloudtest/pkg/tests/shell_provider_test.go (hunk 1 above)
Comment (alex-yust) [reply test/cloudtest/pkg/tests/shell_provider_test.go]: To test a_provider destruction (upon completion of assigned tests, not the whole testing) all tests in a_provider should finish before total tests execution finished (b_provider in out case). Effectively, it means that 'the longest test sequence on a_provider must be less than longest sequence in optimal run on b_provider'

a_provider longest sequence: TestTimeout(5sec) + {TestPass|TesFail}

b_provider longest sequence in 'fastest run' = max(
   TestPass + b_provider.TestDelay(5sec) + TestFail
   TestTimeout(5s) // no TestDelay
)// max

so, TestDelay must be > 5 to ensure the test to pass
currently, it's 7 in my ongoing PR

**Pull request merged**